pdf_melted_app/package.json

```diff
@@ -22,6 +24,8 @@
     "react-dom": "^16.8.4"
   },
   "devDependencies": {
+    "cross-env": "^5.2.0",
+    "crossenv": "^0.0.2-security",
```

Comment:
You'll want to remove this too--forgot to remove it!

Reply:
I have moved it up into default dependencies that (npm install; npm run build) will run after checkout.